go.mod

```diff
@@ -77,7 +75,11 @@ require (
 	upper.io/db.v3 v3.6.3+incompatible
 )
 
-require github.com/go-git/go-git/v5 v5.3.0
+require (
```

Comment:
I think this should be in the require block above instead of separate.

Reply:
Sure, I'll do so. The above go.mod related changes were the magic of go mod tidy :)